### PR TITLE
make converter compatible for files from NASA CSDAP

### LINKED ISSUE #1117: gnssro decode adapt for Spire CSDAP
there is a subtle difference in the files from the NASA CSDAP of Spire data.  This separates from all other bufr encoded sensors .  the change appears to be minor

### PR DESCRIPTION
**Pull request by BenjaminRuston**
## Description

allow converter to read the NASA CSDAP files. The do not contain L1 and L2 frequencies separately along with combined but just a single values. Likely combined when possible.

 
### Issue(s) addressed

- fixes #1117 

## Acceptance Criteria (Definition of Done)

can produce output from both routinely produced bufr by EUMETSAT, CDAAC etc. and those from NASA CSDAP

### CODE REVIEW

**Review by huishao-r — APPROVED** [review]
didn't try with a real obs file. But the code changes make sense to me.

**Comment by BenjaminRuston** [comment]
@fabiolrdiniz we should talk about how to log these converters being updated while the conventions have not taken

**Comment by BenjaminRuston** [comment]
@fabiolrdiniz agree this converter needs an update plan or an additional associated PR which uses the new conventions

**Comment by fabiolrdiniz** [comment]
@BenjaminRuston, sure!

File: src/gnssro/gnssro_bufr2ioda.py [hunk 1]
```diff
@@ -268,7 +274,12 @@ def get_obs_data(bufr, profile_meta_data, add_qc, record_number=None):
 
 def quality_control(profile_meta_data, heights, lats, lons):
 
-    good = (heights > 0.) & (heights < 100000.) & (abs(lats) < 90.) & (abs(lons) < 360.)
+    try:
+        good = (heights > 0.) & (heights < 100000.) & (abs(lats) < 90.) & (abs(lons) < 360.)
```
Comment (fabiolrdiniz) [inline src/gnssro/gnssro_bufr2ioda.py:278]:
```suggestion
        good = (heights > 0.) & (heights < 100000.) & (abs(lats) <= 90.) & (abs(lons) <= 360.)
```

**Review by fabiolrdiniz — APPROVED** [review]
Thanks, @BenjaminRuston!

**Pull request merged**